docs/quickstart.md

```diff
@@ -0,0 +1,176 @@
+the service account that you just created listed under "OAuth 2.0 client IDs". Copy the "Client ID",
+as it will be used in the next step.
+
+#### Authorizing use of the Admin SDK API
+
+Click [here](https://console.cloud.google.com/apis/library/admin.googleapis.com) to go to Google's
+page for the "Admin SDK" API. Click "Enable" to enable the API for your project. Note that if the
+API has already been enabled on your project, then you will see the word "API enabled" along with
+a green checkmark.
+
+![Admin API](images/setup-admin_api.jpg)
+
+Go to the [Google Admin console](https://admin.google.com); note that you may be prompted to log
+in again. Select "Security" from the available controls (click "More Controls" if it is not visible
+when the page first loads).
+
+![Security Admin Control](images/setup-security_control.jpg)
+
+Click the "Advanced settings" option, and select "Manage API client access" from the resulting
+panel.
+
+![API Client Access](images/setup-api_client_access.jpg)
+
+In the "Client Name" field, enter the Client ID of the service account that you have created. You
+will need to provide the following "API Scopes" for the service account that we have created:
+- `https://www.googleapis.com/auth/admin.directory.group.readonly`
+- `https://www.googleapis.com/auth/admin.directory.user.readonly`
+
+Type these into the field titled "One or More API Scopes", as a comma-separated list. After these
+have been filled in, click the "Authorize" button.
+
+#### Configuring `sso` to act as an administrator
+
+When running the `sso-authenticator` binary, the following environment variables must be set, in
+order to act as an administrator and access user group membership data:
+- **`BUZZFEED_GOOGLE_ADMIN_EMAIL`**: An administrative email address on your organization's
+domain, the identity of which can be assumed by `sso`.
+- **`BUZZFEED_GOOGLE_SERVICE_ACCOUNT_JSON`**: A permanent location for the private key `.json` file 
+that was downloaded through your browser at the time of service account creation. There is no
+reason why this file should ever be accessed by any person or service other than `sso`; ensure that
+file permissions are set accordingly.
+
+If you already have an instance of `sso` running, then it will need to be restarted before these
+changes will take effect.
+
+## Docker and Docker Compose Provisioning
+
+Next, third party dependencies must be installed and configuration files modified:
+
+Setup Docker and Docker Compose on your machine by following the directions at https://docs.docker.com/compose/install/
+
+Configure SSO
+ 
+<flesh this out>
+
+
+## Run the containers
+
+Four containers are in the quickstart docker-compose.yml included in this repo. They are `sso_auth` and `sso_proxy`
+containers, which run the `sso` code. A `nginx-proxy` container which acts as a dynamic proxy front-ending the cluster,
+and lastly a `payload` container which runs the payload that's `sso` is providing auth services for.
```

Comment:
I think this is too much detail for a quickstart.  All of this information can be gleaned from the docker-compose.yml file, right?

If we do want to provide this explanatory context, let's move it down below, _after_ the user has hopefully gotten the services up and running using `docker-compose`!